sources/platform/actors/development/programming_interface/environment_variables.md

```diff
@@ -14,6 +14,21 @@ import TabItem from '@theme/TabItem';
 
 ---
 
+## How to use environment variables in Actors
```

Comment:
Use singular in the section. 

Both options set Actor env variables for a single Actor, not multiple. Therefore, using the plural doesn't sound right.